src/Resources/public/admin/js/cmf.js

```diff
@@ -854,9 +854,12 @@ app.SearchFilter.DateRangePicker = (function () {
     $(document).ready(function() {
         window.app.init($('body'));
 
-        $('#customerDuplicatesMerge').on('click', function () {
-            const duplicateIds = $('#customerDuplicatesMerge').data('duplicate-ids');
-            new window.top.pimcore.plugin.objectmerger.panel(duplicateIds[0], duplicateIds[1]);
+        const duplicates = $('.customer-duplicates-merge');
+        $.each(duplicates , function(index,duplicate) {
+            duplicate.addEventListener('click', (event) => {
+                const duplicateIds = JSON.parse(duplicate.dataset.duplicateIds);
```

Comment:
Because 'duplicate-ids' is converted to 'duplicateIds' by DOM API. https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset#in_javascript